bindings/python/utils/std-aligned-map.hpp

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (c) 2019 LAAS-CNRS
+//
+
+#ifndef __pinocchio_python_utils_std_aligned_map_hpp__
+#define __pinocchio_python_utils_std_aligned_map_hpp__
+
+#include <boost/python.hpp>
+#include <boost/python/suite/indexing/map_indexing_suite.hpp>
+#include <string>
+#include <map>
+
+namespace pinocchio
+{
+  namespace python
+  {
+    
+    namespace bp = boost::python;
+    
+    ///
+    /// \brief Expose an std::map from a type given as template argument. Uses the Eigen::aligned_allocater in the value.
+    ///
+    /// \tparam KeyType, ValType Type to expose as std::map<KeyType, ValType, Eigen::aligned_allocator<KeyType, ValType>>.
+    ///
+    /// \sa StdAlignedMapPythonVisitor
+    ///
+    template<class KyeType, class ValType, bool NoProxy = true>
+    struct StdAlignedMapPythonVisitor
```

Comment:
As it is clearly similar to the not aligned version, you should put bot( in the same .hpp file

Reply:
I am following the same standard as std-aligned-vector and std-vector.
Would you still prefer that I change this?

Reply:
I've replied to this issue bellow

Reply:
I just created the python visitor for the sake of completion of the code.
I removed the visitors and the aligned-allocator. the binding is now directly using the bp::map_indexing class